tests/asserts/keywords.py

```diff
@@ -372,8 +475,97 @@ def process_strats(src, paramData):
+                                                msg = f"🔴ERROR: Invalid Home Node:{enemyGroupRef}:{homeNodeRef}"
+                                                messages["reds"].append(msg)
+                                                messages["counts"]["reds"] += 1
+                                    if "betweenNodes" in enemy:
+                                        for betweenNode in enemy["betweenNodes"]:
+                                            betweenNodeRef = f"Node[{roomRef}:{betweenNode}]"
+                                            if betweenNode not in roomData["nodes"]["froms"]:
+                                                msg = f"🔴ERROR: Invalid Between Node:{enemyGroupRef}:{betweenNodeRef}"
+                                                messages["reds"].append(msg)
+                                                messages["counts"]["reds"] += 1
+
+                            # Validate Obstacles
+                            obstacleErrors = search_for_valid_keyvalue(
+                                ["obstaclesCleared", "obstaclesToAvoid"],
+                                "room",
+                                roomData["obstacles"]["ids"],
+                                room
+                            )
+                            if obstacleErrors:
+                                for obstacleError in obstacleErrors:
+                                    msg = f"🔴ERROR: Invalid Obstacles ID:{roomRef}:{obstacleError}"
+                                    messages["reds"].append(msg)
+                                    messages["counts"]["reds"] += 1
+
+                            # Validate Reusable Roomwide Strats
+                            reusableErrors = search_for_valid_keyvalue(
+                                [
+                                    "reusableRoomwideNotable"
+                                ],
+                                "room",
+                                roomData["reusableStrats"]["names"],
+                                room
+                            )
+                            if reusableErrors:
+                                for reusableError in reusableErrors:
+                                    msg = f"🔴ERROR: Invalid Reusable Strat Name:{roomRef}:{reusableError}"
+                                    messages["reds"].append(msg)
+                                    messages["counts"]["reds"] += 1
+
+                            # Validate Requires Nodes
+                            requiresErrors = search_for_valid_keyvalue(
+                                [
+                                    "fromNode",
+                                    "fromNodes.",
+                                    "inRoomPath.",
+                                    "resetRoom.nodes.",
+                                    "nodesToAvoid.",
+                                    "itemNotCollectedAtNode",
+                                    "entranceNodes."
```

Comment:
entranceNodes is part of the room description
`rooms: roomEnvironments: entranceNodes: []`

The node value would be a door (way of entering) node for the room.

Reply:
I've moved `entranceNodes` to it's own check and have it check against nodes that have `nodeType` `door` or `entrance`.
It'll be included in an upcoming commit.